pages/deployment/multicontainer.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Multicontainer Guide
+excerpt: A guide to building multicontainer applications with resin.io
+---
+
+# Multicontainer Guide
+
+As your applications grow more complex, you may find significant benefit in running some services in separate containers. Multicontainer applications allow you to better isolate, duplicate, and manage key services, making your applications more modular and secure. Starting with resinOS vx.x.x, resin.io provides multicontainer application support, built around the [Docker Compose][docker-compose] tool. This guide will cover the considerations you need to take into account when using a multicontainer instance, including Docker Compose configuration, important resin.io specific settings, and single container applications.
+
+## Docker Compose
+
+At the root of your multicontainer application, you'll use a `docker-compose.yml` file to specify the configuration of your containers. The `docker-compose.yml` defines the services you'll be building, as well as how the services interact with each other and the host OS.
+
+Here's an example `docker-compose.yml` for a [simple web server][simple-web-server]:
+
+```
+ version: '2'
+ services:
+   web:
+     build: ./web
+     ports:
+      - "80:5000"
+     depends_on:
+      - redis
+   redis:
+    build: ./redis
+```
+
+Each service can either be built from a directory containing a Dockerfile, as shown here, or can use a **Docker** image that has already been built, by replacing `build:` with `image:`. If your containers need to started in a specific order, make sure to use the `depends_on:` setting.
```

Comment:
It would be nice to link to the `depends_on` docs https://docs.docker.com/compose/compose-file/compose-file-v2/#depends_on